CCT_cold/experiment.js

```diff
@@ -0,0 +1,665 @@
+	firstLoss=0
+	for(z=0; z<randomChosenCards.length; z++){
+	index=whichLossCards.indexOf(randomChosenCards[z])
+		if(index==-1){
+		roundPoints=roundPoints+gainAmt
+		}	
+		else if(index!=-1){
+			if(firstLoss==0){
+		 		roundPoints=roundPoints-lossAmt
+		 		firstLoss=1
+		 	}else if(firstLoss==1){
+		 		roundPoints=roundPoints
+		 	}
+		
+		}
+	}
+	roundPointsArray.push(roundPoints)
+	alert('You chose '+currID+' cards. When you click on ok, the next round starts. Please note that the loss amount, the gain amount, and the number of loss cards might have changed.')		
+		var e = jQuery.Event("keydown");
+		e.which = 37; // # Some key code value
+		e.keyCode = 37
+		$(document).trigger(e);
+		var e = jQuery.Event("keyup");
+		e.which = 37; // # Some key code value
+		e.keyCode = 37
+		$(document).trigger(e)
+}
+
+var chooseButton1 = function(clicked_id){
+	currID = parseInt(clicked_id)
+	alert('You chose '+currID+' cards. When you click on ok, the next round starts. Please note that the loss amount, the gain amount, and the number of loss cards might have changed.')		
+		var e = jQuery.Event("keydown");
+		e.which = 37; // # Some key code value
+		e.keyCode = 37
+		$(document).trigger(e);
+		var e = jQuery.Event("keyup");
+		e.which = 37; // # Some key code value
+		e.keyCode = 37
+		$(document).trigger(e)
+}
+
+var instructButton = function(clicked_id){
+	currID = parseInt(clicked_id)
+	document.getElementById(clicked_id).src='/images/chosen.png';
+}
+
+// appends text to be presented in the game
+function appendTextAfter(input,search_term, new_text) {
+	var index = input.indexOf(search_term)+search_term.length
+	return input.slice(0,index) + new_text + input.slice(index)
+}
+
+
+
+// this function sets up the round params (loss amount, gain amount, which ones are loss cards, initializes the array for cards to be clicked, )
+var getRound = function(){
+	roundPoints=0
+	cardArray=[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32]
+	shuffledCardArray=jsPsych.randomization.repeat(cardArray,1)
+	shuffledCardArray2=jsPsych.randomization.repeat(cardArray,1)
+	whichRound=whichRound+1
+	randomChosenCards=[]
+	roundParams = shuffledParamsArray.pop()
+	lossProb=roundParams[0]
+	gainAmt=roundParams[1]
+	lossAmt=roundParams[2]
+		whichLossCards=[]
+		for(i=0; i<lossProb; i++){
+		whichLossCards.push(shuffledCardArray.pop())
+		}
+	gameState=gameSetup
+		gameState = appendTextAfter(gameState, 'Game Round: ', whichRound)
+		gameState = appendTextAfter(gameState, 'Loss Amount: ', lossAmt)
+		gameState = appendTextAfter(gameState, 'Current Round: ', 0) //make a function to find cumulative score for this round
+		gameState = appendTextAfter(gameState, '# of Loss Cards: ', lossProb)
+		gameState = appendTextAfter(gameState, 'Gain Amount: ', gainAmt)
+	return gameState
+}
+
+
+var instructFunction = function(){
+	document.getElementById("c1").src='/images/chosen.png';
+	document.getElementById("c17").src='/images/chosen.png';
```

Comment:
This hard-coded images path likely will not work in the infrastructure - the path must be to /static/experiments/[experimentname]/images